src/main/java/br/com/openenade/api/nota/NotaController.java

```diff
@@ -58,15 +55,9 @@ public void postNota(@Valid @RequestBody Nota nota) {
     }
 
     @GetMapping("/filterby")
-    public Collection<Nota> getFilteredNotas(@RequestParam("beginAno") Integer beginAno,
-            @RequestParam("endAno") Integer endAno, @RequestParam("categoria") CategoriaAdmin catAdm,
-            @RequestParam("curso") Long curso, @RequestParam("estado") String estado,
-            @RequestParam("modalidade") Modalidade modalidade,
-            @RequestParam("municipio") Long municipio, @RequestParam("regiao") String regiao,
-            @RequestParam("universidade") Long universidade) {
+    public Collection<Nota> getFilteredNotas(NotaFilterInterface notaFilterInterface) {
```

Comment:
@Icaro-Lima you cannot replace all of these parameters with `NotaFilterInterface` because we need to specify the annotations @RequestParam for each parameter.